cluster/swarm/cluster.go

```diff
@@ -231,6 +232,57 @@ func (c *Cluster) Pull(name string, callback func(what, status string)) {
 	}
 }
 
+// Load image
+func (c *Cluster) Load(imageReader io.Reader, callback func(what, status string)) {
+	size := len(c.engines)
+	done := make(chan bool, size)
```

Comment:
Would you mind using a `WaitGroup`, like in `Pull()` (we just changed that because apparently it's a best practice):

https://github.com/docker/swarm/blob/master/cluster/swarm/cluster.go#L213